gdx/src/com/badlogic/gdx/graphics/g2d/Batch.java

```diff
@@ -154,14 +154,16 @@ public void draw (Texture texture, float x, float y, float width, float height,
 	/** Draws a rectangle with the bottom left corner at x,y and stretching the region to cover the given width and height. The
 	 * rectangle is offset by originX, originY relative to the origin. Scale specifies the scaling factor by which the rectangle
 	 * should be scaled around originX, originY. Rotation specifies the angle of counter clockwise rotation of the rectangle around
-	 * originX, originY. */
+	 * originX, originY. 
+	 * @param rotation rotation in degrees */
 	public void draw (TextureRegion region, float x, float y, float originX, float originY, float width, float height,
 		float scaleX, float scaleY, float rotation);
 
 	/** Draws a rectangle with the texture coordinates rotated 90 degrees. The bottom left corner at x,y and stretching the region
 	 * to cover the given width and height. The rectangle is offset by originX, originY relative to the origin. Scale specifies the
 	 * scaling factor by which the rectangle should be scaled around originX, originY. Rotation specifies the angle of counter
 	 * clockwise rotation of the rectangle around originX, originY.
```

Comment:
Not a hard requirement, but it would be nice if the descriptive text also mentioned that the angle is measured in degrees. This also applies to the other two changed sets of docs.